Lib/logging/config.py

```diff
@@ -726,7 +726,7 @@ def configure_handler(self, config):
                 config['address'] = self.as_tuple(config['address'])
             factory = klass
         props = config.pop('.', None)
-        kwargs = dict([(k, config[k]) for k in config if valid_ident(k)])
+        kwargs = {k: config[k] for k in config if valid_ident(k)}
```

Comment:
Same comment here:
dict((k, config[k]) for k in config if valid_ident(k)).
The proposed change also reads well but I'm not sure whether Vinay would want dict comps in this module.